volunteer.html

```diff
@@ -79,7 +79,7 @@
       </form>
     </li>
   </nav>
-  
+
```

Comment:
My IDE keeps editing the white space no matter how many times I revert these changes. I can open it in a new IDE if it bothers y'all too much. But for the short turnaround, this doesn't bother me.